spec/data-types-and-values.html

```diff
@@ -13,6 +13,21 @@ <h1>The Record Type</h1>
       <p>
         The Record type is used to represent the association of multiple primitive values together. Each pair is regarded as occupying a position within the sequence. These positions are indexed with nonnegative integers. The first pair (if any) is at index 0, the next element (if any) at index 1, and so on. The length of a Record is the number of pairs within it. The empty Record has length zero and therefore contains no elements.
       </p>
+
+      <emu-clause id="sec-recordtypecreate" aoid="RecordTypeCreate">
+        <h1>RecordTypeCreate ( _keysList_, _valuesList_ )</h1>
+        <p>This abstract operation RecordTypeCreate takes a _keysList_ and a _valuesList_ argument. It is used to specify the creation of new Record value types. It performs the following steps when called:</p>
+        <emu-alg>
+            1. Assert: Type(_keysList_) is a List of Strings.
+            1. Assert: Type(_valuesList_) is a List of primitive values.

```

Comment:
`Type()` is for JS values, a List isn't one.
```suggestion
            1. Assert: _keysList_ is a List of Strings.
            1. Assert: _valuesList_ is a List of primitive values.

```